molgenis-js/src/test/java/org/molgenis/js/JsMagmaScriptEvaluatorTest.java

```diff
@@ -505,8 +511,25 @@ public void age()
 		Entity person = new DynamicEntity(personBirthDateMeta);
 		person.set("birthdate", now().atOffset(UTC).toLocalDate());
 
-		Object result = jsMagmaScriptEvaluator.eval("$('birthdate').age().value()", person, 3);
+		Bindings bindings = jsMagmaScriptEvaluator.createBindings(person);
```

Comment:
Is suggest moving the performance test to a separate disabled test case that can be executed manually and actually fail when it doesn't complete on time.

Reply:
agreed